### DISCUSSION TITLE
Getting the city-Country for destination in ats

### DISCUSSION THREAD

**Discussion by 1nf3rn0Tr4v** [sjdawson/TruckSimulatorPlugin#11] [Q&A]
![ats_20240419_160216_00](https://github.com/sjdawson/TruckSimulatorPlugin/assets/149985429/43483e30-7f19-49c2-a8a6-8d4e5052a9f8)
is there a way to get the city and 2 letter state like it shows in game?

**Comment by sjdawson** [comment]
Hi @1nf3rn0Tr4v,

There's currently no way (that I can see based on my very limited ATS exposure) to get that information from the data that ATS sends out, though it would still likely be possible with some custom look up tables to fetch the relevant 2 letter state based on the city - would need to generate a list of all the cities in ATS first though!

Regards,

**Reply by 1nf3rn0Tr4v** [reply]
Thank you for your replay I have figured out a way to get it to work it is just a long process typing it in. love the plugin thank you for your hard work on it.

**Comment by NigeUK** [comment]
I was intrigued by this question and went to have a look myself.
I created a dash for ATS and ETS2, so I've explored quite a lot the telemetry through SimHub (and Mr Dawson's excellent plugin).
I can't find any reference to the State that is attached to a job (The state is "Country" in ATS I think).
Never say never though! One of these days it may be available to us!

**Reply by 1nf3rn0Tr4v** [reply]
Thank you for your reply I have come up with a way to show the 2 letter state using a if with return

**Reply by NigeUK** [reply]
Excellent! Which property do you use to determine the state in the first place?

**Reply by 1nf3rn0Tr4v** [reply]
GameRawData.JobValues.CityDestinationId that way any cities that have the same names will show how the game has them
